packages/pds/src/app-view/subscription/repo.ts

```diff
@@ -0,0 +1,187 @@
+import { CID } from 'multiformats/cid'
+import { AtUri } from '@atproto/uri'
+import { wait } from '@atproto/common'
+import { DisconnectError, Subscription } from '@atproto/xrpc-server'
+import {
+  MemoryBlockstore,
+  MST,
+  WriteOpAction,
+  parseDataKey,
+  readCarWithRoot,
+  def as repoDef,
+} from '@atproto/repo'
+import { PreparedWrite } from '../../repo'
+import { OutputSchema as Message } from '../../lexicon/types/com/atproto/sync/subscribeAllRepos'
+import { lexicons } from '../../lexicon/lexicons'
+import Database from '../../db'
+import AppContext from '../../context'
+import { Leader } from '../../db/leader'
+import { appViewLogger } from '../logger'
+
+const METHOD = 'com.atproto.sync.subscribeAllRepos'
```

Comment:
you can make this typesafe by importing `ids` from `lexicon/lexicons` & using `ids.ComAtprotoSyncSubscribeAllRepos`

Reply:
Ah nice, forgot about that!